php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -322,6 +322,7 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
      *                            registered.
      *
      * @return boolean true if the element passes, false if it fails a rule.
+     * @throws \LorisException
```

Comment:
All others `@throws` do not have a `\`. Should all be the same
@driusan Do we need to put a `\` in front on the exception?

Reply:
`\` is required if you're in a namespace, optional if you're in the root namespace..